### ISSUE TITLE
UI Automation Specification - useless page?

### ISSUE THREAD

**Issue by MSDN-WhiteKnight** [dotnet/docs#16887]
https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-specification-and-community-promise 

This page seems to be useless. It just repeats the information from overview. Both links lead to some generic site about accessibility, there's no actual specification, nor community promise. Maybe it should be deleted?

**Comment by gewarren** [comment]
@MSDN-WhiteKnight Agreed, I think this page can be deleted. The [overview page](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-overview) can be added as a redirect target.
